src/operations.cpp

```diff
@@ -613,12 +613,12 @@ namespace
   }  
   
   typedef bool (*Ptr_equal_string_ordinal_ic)(const wchar_t*, const wchar_t*);
-
-  Ptr_equal_string_ordinal_ic equal_string_ordinal_ic = 
-    rtl_equal_unicode_string_api ? equal_string_ordinal_ic_1 : equal_string_ordinal_ic_2;
   
   perms make_permissions(const path& p, DWORD attr)
   {
+    static Ptr_equal_string_ordinal_ic equal_string_ordinal_ic =
```

Comment:
Statics aren't thread safe *period*. I expect to see some synchronization happening here if this needs to be thread safe.